src/main/java/com/twilio/oai/common/EnumConstants.java

```diff
@@ -82,12 +82,20 @@ public enum RubyDataTypes implements LanguageDataType {
 
         private final String value;
     }
-    
+
     @Getter
     @RequiredArgsConstructor
     public enum PredefinedJavaEnumsFormat {
         HTTP_METHOD("http-method");
-        
+
+        private final String value;
+    }
+
+    @Getter
+    @RequiredArgsConstructor
+    public enum PhpDataTypes implements LanguageDataType {
+        LIST("[]");
```

Comment:
Going forward we might be getting Set as well, No action required now.
Adding this comment as a reminder for all helper libs.